NEWS.md

```diff
@@ -1,8 +1,25 @@
+# CHANGES IN GGIR VERSION 3.1-?
+
+- Part 4: 
+
+  - Identification of daylight saving time days in the detection of the spt is improved as it had the risk to misclassify partial first/last days as daylight saving time.
```

Comment:
This only affects the last day, correct? If it also affects the first day shouldn't object `t0` be extracted with minutes and seconds too?